soap_test.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Try to dump SOAP vectors (features) for each atom of each molecule
+
+import sys
+import numpy as np
+from ase.io import read as ase_read
+from rascal.representations import SphericalInvariants as SOAP
+
+# What is this integer? Unit? Reasonable range to explore?
+max_radial = 6
+# What is this integer? Unit? Reasonable range to explore?
```

Comment:
that's the number of spherical harmonics being used for the basis, i.e. the angular resolution. Typical range is 4-8, you can go higher if you want/need